samples/cpp/image_generation/benchmark_text2image.cpp

```diff
@@ -0,0 +1,106 @@
+// Copyright (C) 2023-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#include "openvino/genai/image_generation/text2image_pipeline.hpp"
+#include <cxxopts.hpp>
+#include "imwrite.hpp"
+
+int main(int argc, char* argv[]) try {
+    cxxopts::Options options("benchmark_image_generation", "Help command");
+
+    options.add_options()
+    ("m,model", "Path to model and tokenizers base directory", cxxopts::value<std::string>()->default_value("."))
```

Comment:
please, don't imply that model by default is located in current folder.
I would make this parameters as required as in other benchmark applications:
https://github.com/openvinotoolkit/openvino.genai/blob/4fb48deed27d23cbb517eb1247e8f1f48bdc8596/samples/cpp/text_generation/benchmark_genai.cpp#L10-L11